config/settings.yml

```diff
@@ -838,6 +839,7 @@ vanotify:
     connection_template: fake_template_id
     disconnection_template: fake_template_id
     covid_vaccine_registration: fake_template_id
+    covid_vaccine_confirmation: fake_template_id
```

Comment:
You can remove this since it's the obsolete settings location and just use the new preferred location.